channeldb/graph.go

```diff
@@ -118,6 +118,18 @@ var (
 	// edge's participants.
 	zombieBucket = []byte("zombie-index")
 
+	// disabledEdgePolicyBucket is a sub-bucket of the main edgeBucket bucket
+	// responsible for maintaining an index of disabled edge policies.
+	// Each entry exists within the bucket as follows:
+	//
+	// maps: <chanID><direction> -> nil
```

Comment:
as per my other comment, `nil` here should become `[]byte{}`

Reply:
Fixed.